src/client/push-client.js

```diff
@@ -211,6 +213,9 @@ export default class PushClient extends EventDispatch {
    */
   getSubscription() {
     return this.getRegistration()
+    .catch(() => {
```

Comment:
Similar to above.

Hmm. Though I guess that this is masking errors, actually. Do we need to have the `catch` in these places at all? It seems that if we are not handling the errors meaningfully then we should let the user `catch` them.